apps/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "lint": "yarn lint:scss && yarn lint:js",
     "lint:fix": "yarn lint:scss --fix && yarn lint:js --fix",
-    "lint:js": "eslint --format ./.eslintCustomMessagesFormatter.js --ext=.js,.jsx,.ts,.tsx .",
+    "lint:js": "eslint --format ./.eslintCustomMessagesFormatter.js --ext=.js,.jsx,.ts,.tsx --report-unused-disable-directives-severity=error .",
```

Comment:
another more general option here would be `--max-warnings 0`: https://eslint.org/docs/latest/use/command-line-interface#--max-warnings